src/Sulu/Bundle/ContentBundle/Tests/app/AppKernel.php

```diff
@@ -18,6 +18,10 @@ public function registerContainerConfiguration(LoaderInterface $loader)
     {
         parent::registerContainerConfiguration($loader);
 
+        if (getenv('SYMFONY__PHPCR__TRANSPORT') === 'jackrabbit') {
```

Comment:
I don't have a better (and fastly implementable) idea how to solve this. It's about enabling versioning only for jackrabbit... A solution might be to add another environment variable. Would allow us to also test jackrabbit without versioning. Then we would have another build more, and the tests are taking longer, ... I think I would leave it like that.